### ISSUE TITLE
Lita won't respond to self.

### ISSUE THREAD

**Issue by MrJoy** [litaio/lita-hipchat#35]
When I have Lita connect to HipChat as me, it does not recognizes messages I send when I log in from web/native client.

It makes it hard to develop handlers if I have to have multiple accounts to be able to talk to the bot.  Could a flag be added to allow it to listen to messages from its own user ID?

**Comment by jimmycuadra** [comment]
No, because this could cause infinite loops. I'm afraid using a dedicated account for the bot, even during testing, is the best approach. I test plugins on HipChat that way, too.

**Comment by MrJoy** [comment]
I understand the risk, but a second user is simply not possible in some contexts such as my current one:  Access is gated by IT department and mechanisms don't exist for creating users not corresponding to a human.

**Comment by jimmycuadra** [comment]
How is Lita deployed in production if all HipChat accounts need to correspond to a real person? Or do you mean that you're not allowed to create a secondary test account for it?

**Comment by MrJoy** [comment]
It's not presently deployed to production.  I'm building out a Lita setup as a proof-of-concept.

**Comment by MrJoy** [comment]
(I.E. I might be able to convince folks it's worth it to expand the process if, and only if, the organization sees value in the bot.  Essentially, that can't happen until the bot exists.)

**Comment by jimmycuadra** [comment]
Ah, I see. I think this use case is unusual enough that I don't want to add any configuration to disable the no-messages-from-self behavior. Two workarounds for you I can think of: set up a test HipChat account where you have full admin access, and if that's not feasible because the plugins you're developing require access to company resources, use your own fork of Lita that has the check for messages-from-self removed.
